Mathlib/Order/Partition/Finpartition.lean

```diff
@@ -186,6 +205,10 @@ theorem parts_nonempty (P : Finpartition a) (ha : a ≠ ⊥) : P.parts.Nonempty
   parts_nonempty_iff.2 ha
 #align finpartition.parts_nonempty Finpartition.parts_nonempty
 
+@[simp]
+theorem parts_map {β : Type*} [Lattice β] [OrderBot β] {e : α ≃o β} :
+    (P.map e).parts = P.parts.map e := rfl
```

Comment:
This should be just after the definition of `map`

Reply:
Done.